.github/dependabot.yaml

```diff
@@ -0,0 +1,35 @@
+# Configuration: https://dependabot.com/docs/config-file/
+# Docs: https://docs.github.com/en/github/administering-a-repository/keeping-your-dependencies-updated-automatically
+
+version: 2
+
+updates:
+  - package-ecosystem: "pip"
```

Comment:
We can also have it check the npm package ecosystem for the cognee frontend

Reply:
We can, but Boris is not maintaining it. So, for now should be paused I think

Reply:
@coderabbitai review